CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ endif()
 # C++14 is required
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_EXTENSIONS ON)
```

Comment:
ON is the default (could be removed).

Reply:
I think it's better to be explicit in this case so we know what's going on.

Reply:
I agree that being explicit here is better. Also makes git blame easier when the line is still there.